lms/static/sass/base/_variables.scss

```diff
@@ -82,6 +82,28 @@ $gray-d2: shade($gray,40%); // #4c4c4c
 $gray-d3: shade($gray,60%); // #323232
 $gray-d4: shade($gray,80%); // #191919
 
+// TO-DO: once existing lms $blue is removed, change $cms-blue to $blue. 
+$cms-blue: rgb(0, 159, 230);
+$blue-l1: tint($cms-blue,20%);
```

Comment:
Cool, thanks for the 411. Just checking to make sure your stuff wouldn't be wrecked and catch the $blue blues later in the compile.